data/helpers.d/utils

```diff
@@ -24,3 +24,86 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Download and uncompress the source from app.src
+#
+# The file conf/app.src need to contains:
+# 
+# src_url="Address to download the app archive"
+# src_sum="Control sum"
+# src_format="tar.gz" # (Optional) default value: tar.gz
+# src_in_subdir=false # (Optional) Put false if source are directly in the archive root
+# src_filename="example.tar.gz" (Optionnal) default value: ${src_id}.${src_format}
+#
+#
+# usage: ynh_setup_source dest_dir [source_id]
+# | arg: dest_dir  - Directory where to setup sources
+# | arg: source_id - Name of the app, if the package contains more than one app
+ynh_setup_source () {
```

Comment:
It's not really just a download and an unpack.
This helper, as it is now, unpack directly in the directory.
So it's more like a whole setup for a webapp.